<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="startAutomaticallyCheckbox">
     <property name="text">
      <string>&amp;Start automatically</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="disconnectOtherLocationsCheckbox">
     <property name="text">
      <string>&amp;Disconnect this client if I log in from another location (and vice versa)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="enableLocalHistoryCheckbox">
       <property name="text">
        <string>Store chat &amp;history on this computer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deleteHistoryButton">
       <property name="text">
        <string>Delete history</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="alwaysDisplayOfflineFriendsCheckbox">
     <property name="text">
      <string>Always display &amp;offline friends</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="submitUsageStatsCheckbox">
     <property name="text">
      <string>Help to &amp;improve Hyves Desktop by submitting usage statistics</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
